### DISCUSSION TITLE
Steering Committee - Meeting Minutes and Decisions

### DISCUSSION THREAD

**Discussion by binste** [vega/altair#3164] [General]
We'll use this discussion to document decisions taken in meetings of the [Vega-Altair Steering Committee](https://github.com/altair-viz/altair/blob/main/governance/org-docs/STEERING-COMMITTEE.md). Everyone is welcome to provide inputs by replying to the individual discussion comments. Top-level comments are reserved for the minutes.

**Comment by binste** [comment]
# 2023-08-25
All committee members were present:
* @joelostblom 
* @mattijn 
* @jonmmease 
* @ChristopherDavisUCI 
* @binste 

# Decisions and next actions
All decisions were made by consensus. No vote was necessary.

* @binste will act as steering committee chair. The responsibilities include organising committee activities such as meetings and votes as outlined in [CHARTER.md](https://github.com/altair-viz/altair/blob/main/governance/org-docs/CHARTER.md). He will record decisions and next actions in this GitHub discussion
* [Vega conference](https://idl.cs.washington.edu/vega-conf/):
  * We plan to give the following lightning talks, although they are subject to change:
    * @binste: altair_tiles
    * @ChristopherDavisUCI: How he uses Altair in education / Heatmaps in Altair & Vega-Lite
    * @jonmmease: vl-convert
    * @mattijn: ["Algebra syntax"](https://github.com/altair-viz/altair/discussions/3125#discussioncomment-6571911)
  * We plan to give a general update on the state of Altair. @jonmmease will share a link to a Google Sheets deck based on a previously prepared lightning talk. It's planned that the talk will be given by the committee members who attend the conference in-person. We currently aim to provide updates on the following topics with the person preparing the slides tagged next to it:
    * Recent features in versions 5.0 and 5.1 @joelostblom 
    * Governance structure and code of conduct @binste 
    * Roadmap @jonmmease 
  * Version 5.1 release
    * We still want to include #3163 
    * Release planned for Monday. Release manager is @mattijn 
 * We are considering to apply to become a [Numfocus sponsored project](https://numfocus.org/sponsored-projects). This idea will be further discussed at the Vega conference after which we plan to take a decision on this. Next [application deadline is October 15th](https://numfocus.org/projects-overview).
 * @binste will organise the next meeting for a date after the Vega conference

**Reply by joelostblom** [reply]
If I have time to put some slides together, I think I'm going to try to give a lightning talk about altair_ally as well, or at least the motivation for having functionality like that available in Altair. I would have wanted to spend more work on ithe package first, but maybe it's good to just discuss the ideas and get input at this stage.

**Reply by binste** [reply]
Agree, it's a great opportunity to get inputs and I'd definitely be interested in hearing more about altair_ally! I have the same goal with the altair_tiles lightning talk and I think it's totally fine if the package requires more work.

**Comment by binste** [comment]
# 2023-10-13
All committee members were present:
* @joelostblom 
* @mattijn 
* @jonmmease 
* @ChristopherDavisUCI 
* @binste 

# Decisions and next actions
All decisions were made by consensus. No vote was necessary.

* We exchanged impressions from the Vega/Altair conference which took place in September. Based on this, we decided to not yet apply to become a NumFOCUS sponsored projects and first continue the ongoing discussions with the maintainers of Vega and Vega-Lite on a potential joint governance model including an overall steering committee. See also discussions happening in https://github.com/vega/.github/pull/2.
* @binste to organize the next steering committee call where we decided to also invite core Vega and Vega-Lite developers for the purpose of continuing this discussion

**Reply by binste** [reply]
As a follow-up to this meeting, the committee decided unanimously to add the https://github.com/altair-viz/dash-vega-components project to the altair-viz organisation and place it under the same governance structure.

**Comment by binste** [comment]
# 2023-11-28
As decided in the meeting from 2023-10-13, we used this call to discuss a potential joint governance model with Vega and Vega-Lite. For this purpose, @jheer (Vega) and @domoritz (Vega-Lite) joined the call. All Vega-Altair committee members were present except for @mattijn.

Participants:
* @jheer (Vega)
* @domoritz (Vega-Lite)
* @binste 
* @joelostblom 
* @ChristopherDavisUCI 
* @jonmmease 

# Decisions
Decisions are preliminary until @mattijn approves as well. **@mattijn could you please indicate your approval/disapproval of the points below via comment to this post?** Of course, happy to elaborate further if you want or catch up in a call. We tried to take your view into account based on past discussions.

* We will establish a steering committee for all Vega projects including Vega-Altair. This committee will replace the current one for Vega-Altair. Members of the Vega-Altair steering committee become members of the new committee.
* Each project, e.g. Vega-Altair, can continue to have maintainer meetings to discuss project specific topics such as cutting a new release. The steering committee will deal with big picture topics.
* The new steering committee will meet in a cadence of initially every 1-2 months. Later on we might switch to a lower frequency.
* Communication/discussions/decisions will continue to happen on GitHub so that they are public. More lightweight coordination can happen via Slack.
* @domoritz and @binste will act as committee chairs
* Regarding the NumFOCUS application, there is a general consensus that this is worth pursuing for Vega-Altair. If NumFOCUS is ok with it, it would be great to join as a unified Vega organisation as features in Vega-Altair will often depend on modifications in Vega-Lite and Vega -> Would be great if we can direct funding to the relevant project and not just to Vega-Altair. No decision was made if we want to apply already in January.
  * For the application, and to make it easier to collaborate across projects, the participants discussed the idea again to move Vega-Altair into the Vega GitHub organisation. This can be picked up in a future committee call. We'll need to think through how to solve redirects from current Altair documentation to a new one or if we want to keep the current repo just for the documentation.


# Next actions
* @domoritz: Create PR to move steering committee content from Altair to Vega level: https://github.com/altair-viz/altair/pull/3277 and https://github.com/vega/.github/pull/2
  * @binste: Ensure that Altair docs page is also updated accordingly
* @domoritz to reach out to Jake regarding permissions on GitHub and Pypi as well as to seek inputs on potentially merging GitHub organisations.
* Next meeting to be scheduled by @binste and @domoritz once new governance structure is in place. For agenda, see point above about merging GitHub organisations.

**Reply by mattijn** [reply]
I approve! 
Happy to join the discussion about potential merging of repos. This has been discussed before without reaching consensus.

**Reply by binste** [reply]
Great to hear, thanks for the prompt feedback. And yes, these discussions should definitely continue with you onboard! Joel already recounted a bit a previous conversation you had. It's probably good to devote a next committee call to this to talk through it in detail.
